lnbits/core/migrations.py

```diff
@@ -404,3 +404,43 @@ async def m016_add_username_column_to_accounts(db):
         await db.execute("ALTER TABLE accounts ADD COLUMN extra TEXT")
     except OperationalError:
         pass
+
+
+async def m017_add_timestamp_columns_to_accounts_and_wallets(db):
+    """
+    Adds created_at and updated_at column to accounts and wallets.
+    """
+    try:
+        if db.is_postgres:
```

Comment:
This sets a bad precedent. All db type decisions should be encapsulated in the `db.py` file.

How about adding this in `db.py`:

```py
    @property
    def timestamp_column_default(self) -> str:
        if self.type in {POSTGRES, COCKROACH}:
            return self.timestamp_now
        return "NULL"
```

Then also allow `NULL` for PostgreSQL (the data model must reflect his anyhow)